Service/ResponseFactory.php

```diff
@@ -16,6 +18,8 @@
 {
     const CODE_WRONG_ARGS = 'GEN-ARGUMENTS';
 
+    const CODE_VALIDATION_FAILED = 'GEN-VALIDATION';
```

Comment:
Hm, isnt this the same as "CODE_WRONG_ARGS"?

Reply:
Its the different response, with errors. That's why I added another internal code. Should I remove it?

Reply:
Im not sure. Lets discuss. When would you use `CODE_WRONG_ARGS`?

Reply:
When I don't care where the validation failed.

Reply:
I.E. apps using API without UI, so validation errors are redundant.

Reply:
I do not understand. In my opinion `CODE_WRONG_ARGS` is the same as `CODE_VALIDATION_FAILED`. You said you disagreed. So: It what scenario would you use `CODE_WRONG_ARGS` where `CODE_VALIDATION_FAILED` did make any sense?

Reply:
I didn't say I disagree, I explained why I added it :).

Reply:
And it made sense to me because response has extra data you can use. Anyway,  don't feel strong about it at all, so I will remove it.

Reply:
I feel that a "validation error" is the same as "wrong arguments"

Reply:
Yep, it is.